src/panels/config/integrations/ha-integration-card.ts

```diff
@@ -260,6 +261,13 @@ export class HaIntegrationCard extends LitElement {
                 "ui.panel.config.integrations.config_entry.delete"
               )}
             </mwc-list-item>
+            ${item.state === "loaded"
```

Comment:
I think we should move this above delete. Delete being at the bottom will cause less accidental clicks

Reply:
I think that we should update the config entry data to include if it is reloadable, so that we don't show it up for entries that don't support it.

We should add that here https://github.com/home-assistant/core/blob/dev/homeassistant/components/config/config_entries.py#L330

Reply:
done.   There is probably a better way to deal with the shifting index, but I'm not familiar enough with mwc to know it.